### DISCUSSION TITLE
Space ROS Technical Meeting Agenda 2024-12-12

### DISCUSSION THREAD

**Discussion by Bckempa** [space-ros/space-ros#238] [General]
Previous Agenda: https://github.com/space-ros/space-ros/discussions/234
Active Milestone: [`jazzy-2025.01.0`](https://github.com/orgs/space-ros/projects/3/views/12)

### Agenda
- [x] Introduce any new attendees.
- [x] Summarize any PRs merged since last meeting.
      - https://github.com/space-ros/space-ros/pull/205#event-15398570285
- [ ] Issue Review 
- [ ] Open next meeting discussion.

**Comment by Bckempa** [comment]
Merge Queue (in the abstract sense) from our [previous discussion](https://github.com/space-ros/space-ros/discussions/233#discussioncomment-11256251):

- https://github.com/space-ros/space-ros/pull/205
- https://github.com/space-ros/space-ros/pull/186
- https://github.com/space-ros/space-ros/pull/151

@Aasimonov any blockers on migrating to jazzy?

**Comment by EzraBrooks** [comment]
Hiro finally got approval to release the RACS2 bridge to Space ROS, let's talk about the logistics

**Comment by Bckempa** [comment]
Next meeting agenda: https://github.com/space-ros/space-ros/discussions/243
